.fuel-core/configs/chainConfig.json

```diff
@@ -7,7 +7,7 @@
           "max_inputs": 255,
           "max_outputs": 255,
           "max_witnesses": 255,
-          "max_gas_per_tx": 100000000,
+          "max_gas_per_tx": 30000000,
```

Comment:
Why is this set to `30000000` as opposed to `100000000` as in the [config](https://github.com/FuelLabs/chain-configuration/blob/master/upgradelog/ignition-dev/consensus_parameters/6.json)?

Reply:
@maschad I am using [this consensus parameters](https://github.com/FuelLabs/chain-configuration/blob/2dbbe4c6b40f9e458d65945a1f6f1e25888cf669/upgradelog/ignition/consensus_parameters/5.json) instead, which is from the latest upgrade for testnet, not for devnet.